src/style.css

```diff
@@ -1 +1,24 @@
-/* add styles here */
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars {
+  display: flex;
+}
+
+.container {
+  margin: 0;
+}
```

Comment:
Avoid setting margins to zero directly on container elements. Consider using a more specific class or a utility class to apply zero margins, ensuring consistency and preventing potential layout issues.